staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -108,7 +132,7 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr admission.Attr
 		go func(invocation *generic.WebhookInvocation, idx int) {
 			ignoreClientCallFailures := false
 			hookName := "unknown"
-			versionedAttr := versionedAttrs[invocation.Kind]
+			versionedAttr, _ := versionedAttrAccessor.VersionedAttributeFromCache(invocation.Kind)
```

Comment:
Probably safest to handle versionedAttr missing from the cache here, although it doesn't look like it would ever happen.